lib/util.js

```diff
@@ -620,7 +620,78 @@ function create_wordpress_paginator(feed) {
     return _doCreateWordpress(feed);
 }
 
+/**
+ * @property {Array} remove - The default values for ....
+ * @property {Array} removeNoText - The default values for ...
+ * @property {Array} removeAttrs - The default values for ...
+ * @property {boolean} noComments - The default values for ...
+ * @memberof util
+ */
+const CLEANUP_DEFAULTS = {
+    remove: [ 'script', 'noscript', 'style', 'iframe', 'hr', 'br + br', 'center' ],
+    removeNoText: ['p'],
+    removeAttrs: [[ ['*'], [ 'style', 'class' ] ]],
```

Comment:
I would suggest making the removeAttrs api to take a dictionary, e,g.:
```js
{'*': ['style', 'class']}
```
I know that would disallow things that are possible with the current API, like `[[['b', 'i']: ['role']]]` but the dictionary style seems to me a lot easier to think about. Arrays of arrays of arrays are difficult to grasp in my experience.
Otherwise, maybe something like this:
```js
[
  { tags: ['b', 'i'], attrs: ['class', 'style'] },
]
```

Reply:
Good point, I will go with the dictionary API. It doesn't disallow multiple tags, because the passed CSS selector can be: `{'b, i': ['style', 'class']}`